<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>962</width>
    <height>485</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>OPC UA Client</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: white;
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QComboBox" name="addressComboBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid black;</string>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="1" colspan="2">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>Ubuntu Mono</family>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Details</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>Ubuntu Mono</family>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" rowspan="6">
     <widget class="QTreeView" name="treeView">
      <property name="styleSheet">
       <string notr="true">border: 2px solid black;
</string>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
     </widget>
    </item>
    <item row="6" column="1" rowspan="6" colspan="2">
     <widget class="QTreeView" name="attrView">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid black;
</string>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>962</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSubscription">
    <property name="title">
     <string>Subscription</string>
    </property>
    <addaction name="actionCreate_subscription"/>
    <addaction name="actionDelete_subscription"/>
    <addaction name="actionCreate_monitored_item"/>
    <addaction name="actionDelete_monitered_item"/>
   </widget>
   <widget class="QMenu" name="menuReadWrite">
    <property name="title">
     <string>ReadWrite</string>
    </property>
    <addaction name="actionRead"/>
    <addaction name="actionWrite"/>
   </widget>
   <widget class="QMenu" name="menuWindows">
    <property name="title">
     <string>Windows</string>
    </property>
    <addaction name="actionReferences"/>
    <addaction name="actionSubscriptions"/>
    <addaction name="actionLogs"/>
   </widget>
   <widget class="QMenu" name="menuConnection">
    <property name="title">
     <string>Connection</string>
    </property>
    <addaction name="actionConnection_settings"/>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
    <addaction name="actionQuery_endpoints"/>
   </widget>
   <addaction name="menuConnection"/>
   <addaction name="menuReadWrite"/>
   <addaction name="menuSubscription"/>
   <addaction name="menuWindows"/>
  </widget>
  <action name="actionRead">
   <property name="text">
    <string>Read</string>
   </property>
  </action>
  <action name="actionWrite">
   <property name="text">
    <string>Write</string>
   </property>
  </action>
  <action name="actionCreate_subscription">
   <property name="text">
    <string>Create subscription</string>
   </property>
  </action>
  <action name="actionDelete_subscription">
   <property name="text">
    <string>Delete subscription</string>
   </property>
  </action>
  <action name="actionCreate_monitored_item">
   <property name="text">
    <string>Create monitored item</string>
   </property>
  </action>
  <action name="actionDelete_monitered_item">
   <property name="text">
    <string>Delete monitered item</string>
   </property>
  </action>
  <action name="actionReferences">
   <property name="text">
    <string>References</string>
   </property>
  </action>
  <action name="actionSubscriptions">
   <property name="text">
    <string>Subscriptions</string>
   </property>
  </action>
  <action name="actionLogs">
   <property name="text">
    <string>Logs</string>
   </property>
  </action>
  <action name="actionConnection_settings">
   <property name="text">
    <string>Connection settings</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionQuery_endpoints">
   <property name="text">
    <string>Query endpoints</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
